### DISCUSSION TITLE
B站直播右下角加了备案号

### DISCUSSION THREAD

**Discussion by yongrenx** [the1812/Bilibili-Evolved#4624] [提问 / Q&A] [answered]
这个可以去除吗
<img width="208" alt="9b21189e46e291fccd74e93e479ae45f" src="https://github.com/the1812/Bilibili-Evolved/assets/120461815/f3ecbbc3-ec09-4174-9a53-30a1b01d2764">

**Comment by the1812** [comment]
没见过, 这真不是主播放的?

**Reply by yongrenx** [reply]
不太清楚，只在英雄联盟直播看见过

**Comment by sino1641** [comment, marked as answer]
体育类直播强加的 全平台都有
